hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -1426,6 +1431,79 @@ public static void invalidateCacheEntry(String cacheKey) {
     }
   }
 
+  public List<String> getCompactionLogSstFiles(Path checkpointPath)
```

Comment:
We can directly pass the checkpoint OMMetadataManager initiazed using
https://github.com/apache/ozone/blob/da030c0fac84123ff7ebd15eb3edbaf6ba0e4aff/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L220

and access compactionLogTable and iterate directly
https://github.com/apache/ozone/blob/da030c0fac84123ff7ebd15eb3edbaf6ba0e4aff/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L1668
which would avoid redundantly initializing this rocksdb open code

Reply:
This logic would be very simplified after that